packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: fivetran/iterable_source
-    version: [">=0.2.0", "<0.3.0"]
\ No newline at end of file
+    version: [">=0.3.0", "<0.4.0"]
```

Comment:
I know this will pass once live, but let's just make sure to hold off releasing this package until the source is released so we can 100% confirm the package passes before merge and release.